README.md

```diff
@@ -4,6 +4,58 @@ Enables Postman support of the WSDL specification
 browserify index.js --standalone xsd2jsonschemafaker > xsd2jsonschemafaker.js
 
 
+## Command Line Interface
+
+The converter can be used as a CLI tool as well. The following [command line options](#options) are available.
+
+`wsdl2postman [options]`
+
+### Options
+- `-v`, `--version`  
+  Specifies the version of the converter
+
+- `-s <source>`, `--spec <source>`  
+  Used to specify the WSDL specification (file path) which is to be converted
+
+- `-o <destination>`, `--output <destination>`  
+  Used to specify the destination file in which the collection is to be written
+
+- `-t`, `--test`  
+  Used to test the collection with an in-built sample specification
+
+- `-p`, `--pretty`  
+  Used to pretty print the collection object while writing to a file
+
+- `-O`, `--options`
+  Used to supply options to the converter
+
+- `-c`, `--options-config`  
+  Used to supply options to the converter through config file
+
+- `-h`, `--help`  
+  Specifies all the options along with a few usage examples on the terminal
+
+
+### Usage
+
+**Sample usage examples of the converter CLI**
+
+
+- Takes a specification (spec.wsdl) as an input and writes to a file (collection.json) with pretty printing and using provided options
+```terminal
+$ wsdl2postman -s spec.wsdl -o collection.json -p -O folderStrategy=Service
```

Comment:
I tried out options, and currently no options are applied while using this syntax. Can you check what's wrong here.

Reply:
Added in last commits, resolved issue and added ut with scenario